root-cert-stats/background.js

```diff
@@ -1,6 +1,6 @@
 "use strict";
 
-let rootCertStats = {};
+var rootCertStats = {};
```

Comment:
```suggestion
// Note: declared with "var" because popup.js references this global variable.
// If this were to be declared with "const" or "let", then the variable would
// still be available to this file, but not to popup.js.
var rootCertStats = {};
```